### PR TITLE
Python SDK: Add 25 second default timeout to requests

### PR DESCRIPTION
**Pull request by awolfden**
## Description
Python SDK: Add 25 second default timeout to requests

## Documentation

Does this require changes to the WorkOS Docs? E.g. the [API Reference](https://workos.com/docs/reference) or code snippets need updates.

```
[ ] Yes
```

If yes, link a related docs PR and add a docs maintainer as a reviewer. Their approval is required.

### CODE REVIEW

File: workos/utils/request.py [hunk 1]
```diff
@@ -95,11 +100,11 @@ def request(
                 raise AuthenticationException(response)
             elif status_code == 403:
                 raise AuthorizationException(response)
-            elif status_code == 404:
```
Comment (PaulAsjes) [inline workos/utils/request.py:98]: Why remove this?

**Review by PaulAsjes — CHANGES_REQUESTED** [review]

File: workos/utils/request.py (hunk 1 above)
Comment (awolfden) [reply workos/utils/request.py:98]: Definitely didn't mean to remove this, fat finger error. Will update.

Comment (awolfden) [reply workos/utils/request.py]: @PaulAsjes updated

File: workos/utils/request.py [hunk 2]
```diff
@@ -99,7 +105,9 @@ def request(
                 raise NotFoundException(response)
             error = response_json.get("error")
             error_description = response_json.get("error_description")
-            raise BadRequestException(response)
+            raise BadRequestException(
+                response, error=error, error_description=error_description
```
Comment (PaulAsjes) [inline workos/utils/request.py:109]: More of a nit, but I wonder why we weren't returning these earlier...

**Review by PaulAsjes — APPROVED** [review]

**Pull request merged**